ext/hal/nxp/imx/Kconfig

```diff
@@ -0,0 +1,38 @@
+# Kconfig - IMX M4 Core SDK
+#
+# Copyright (c) 2016, Freescale Semiconductor, Inc.
```

Comment:
Update the year and use your own copyright. If you want to assign copyright to Freescale/NXP, use the NXP copyright: `Copyright (c) 2018, NXP`